File: src/main/java/io/github/alexiscomete/lapinousecond/worlds/buildings/autorisations/AllAutorisation.java
package io.github.alexiscomete.lapinousecond.worlds.buildings.autorisations;

import io.github.alexiscomete.lapinousecond.entity.Owner;

public class AllAutorisation implements BuildingAutorisation {
    @Override
    public boolean isAutorise(Owner owner) {
        return true;
    }
}

File: src/main/java/io/github/alexiscomete/lapinousecond/worlds/buildings/autorisations/AllAutorisation.kt
package io.github.alexiscomete.lapinousecond.worlds.buildings.autorisations

import io.github.alexiscomete.lapinousecond.entity.entities.Owner

class AllAutorisation : BuildingAutorisation {
    override fun isAutorise(owner: Owner?): Boolean {
        return true
    }
}